pkg/kubectl/util/term/resize.go

```diff
@@ -101,14 +101,8 @@ func (s *sizeQueue) monitorSize(outFd uintptr, initialSizes ...*remotecommand.Te
 				if !ok {
 					return
 				}
+				s.resizeChan <- size
```

Comment:
I don't think this cleanup is valid. When `resizeChan` is full, pass size to the channel could put the current goroutine to sleep.

Reply:
Hi @draveness, thanks a lot for your review I always appreciate it. You are completely correct I missed that part in my logic. I will close this PR!